src/classes/updates.py

```diff
@@ -70,13 +70,13 @@ def json(self, is_array=False, only_value=False):
 
         # Build the dictionary to be serialized
         if only_value:
-            data_dict = copy.deepcopy(self.values)
+            data_dict = json.loads(json.dumps(self.values))
```

Comment:
Possibly make a method that combines the dumps and loads method so you could more easily fix weird copy issues if they come up.

Reply:
Yeah, I considered that - however this is a common pattern, and I kind of want to make it obvious what is happening. But it's a good suggestion.